optaplanner-core/src/main/java/org/optaplanner/core/impl/heuristic/HeuristicConfigPolicy.java

```diff
@@ -144,23 +113,19 @@ public boolean isInitializedChainedValueFilterEnabled() {
     // Builder methods
     // ************************************************************************
 
-    public HeuristicConfigPolicy<Solution_> createPhaseConfigPolicy() {
-        return new HeuristicConfigPolicy<>(environmentMode, logIndentation,
-                moveThreadCount, moveThreadBufferSize, threadFactoryClass,
-                scoreDirectorFactory);
+    public Builder<Solution_> buildClone() {
+        return new Builder<>(environmentMode, moveThreadCount, moveThreadBufferSize, threadFactoryClass, scoreDirectorFactory)
+                .withLogIndentation(logIndentation);
     }
 
-    public HeuristicConfigPolicy<Solution_> createFilteredPhaseConfigPolicy() {
-        HeuristicConfigPolicy<Solution_> heuristicConfigPolicy = createPhaseConfigPolicy();
-        heuristicConfigPolicy.reinitializeVariableFilterEnabled = true;
-        heuristicConfigPolicy.initializedChainedValueFilterEnabled = true;
-        return heuristicConfigPolicy;
+    public HeuristicConfigPolicy<Solution_> createPhaseConfigPolicy() {
+        return buildClone().build();
```

Comment:
Or maybe `clonerBuilder().build()`?

Reply:
Sounds much better, as the `buildClone` method is not creating the clone itself.